packages/core/content-type-builder/admin/src/components/DataManagerProvider/index.js

```diff
@@ -107,8 +109,7 @@ const DataManagerProvider = ({
         { data: reservedNames },
       ] = await Promise.all(
         ['components', 'content-types', 'reserved-names'].map((endPoint) => {
-          // TODO: remember to pass also the pluginId when you use the new get, post, put, delete methods from getFetchClient
-          return axiosInstance.get(endPoint);
+          return fetchClient.get(`/${pluginId}/${endPoint}`);
```

Comment:
is there a reason ? we cannot use `get` like we do the `put,post,del` it would be better if we stay consistent

Reply:
ok got it it's because of `lodash`, nvm